evalit/controller_test.py

```diff
@@ -0,0 +1,69 @@
+# read a yaml file
+import matplotlib.pyplot as plt
+import numpy as np
+import yaml
+from mft.mft_automation import MFTAutomation
+from nifi.nifi_automation import NifiAutomation
+from rclone.rclone_automation import RcloneAutomation
+
+
+def generate_grapgs(title, times):
+    times = times - np.min(times)
+
+    for i in range(len(times)):
+        plt.barh(i + 1, times[i][1] - times[i][0], left=times[i][0])
+
+    plt.savefig(title + ".png")
+
+
+def caclulate_throughput(file_sizes, times):
+    total_valume = 0.0
+    for i in range(len(file_sizes)):
+        total_valume += file_sizes[i]
+    throughput = total_valume * 8 / (np.max(times) - np.min(times))
+    print("Throughput: " + str(throughput))
+
+
+# TODO: Fetch theses values from the S3 python client
+file_sizes = [1]  # GB
+file_list = ["testfile"]  # File list in source bucket
+
+# nifi_installation = "/sproj/MFT/nifi-1.15.3"
+nifi_installation = "/home/nishan/software/nifi/nifi-1.15.3"
```

Comment:
is there a way we can avoid hardcoding this path?

Reply:
For file_list, I am planning to use s3 boto3 to get the list from source. That will be better I think.